financial-connections/src/test/java/com/stripe/android/financialconnections/screenshottests/FinancialConnectionsShotShowkaseScreenshotTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.stripe.android.financialconnections.screenshottests
+
+import androidx.activity.OnBackPressedDispatcher
+import androidx.activity.OnBackPressedDispatcherOwner
+import androidx.activity.compose.LocalOnBackPressedDispatcherOwner
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import com.airbnb.android.showkase.models.Showkase
+import com.airbnb.android.showkase.models.ShowkaseBrowserComponent
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import com.stripe.android.financialconnections.getMetadata
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(TestParameterInjector::class)
+class PaparazziSampleScreenshotTest {
+
+    object PreviewProvider : TestParameter.TestParameterValuesProvider {
+        override fun provideValues(): List<ComponentTestPreview> {
+            val metadata = Showkase.getMetadata()
+            return metadata.componentList.map(::ComponentTestPreview)
+        }
+    }
+
+    @Suppress("Unused")
+    enum class BaseDeviceConfig(
+        val deviceConfig: DeviceConfig,
+    ) {
+        NEXUS_5(DeviceConfig.NEXUS_5),
+        PIXEL_C(DeviceConfig.PIXEL_C),
+    }
+
+    @get:Rule
+    val paparazzi = Paparazzi(
+        maxPercentDifference = 0.0
+    )
+
+    @Test
+    fun preview_tests(
+        @TestParameter(valuesProvider = PreviewProvider::class) componentTestPreview: ComponentTestPreview,
+        @TestParameter baseDeviceConfig: BaseDeviceConfig,
```

Comment:
Might be a good idea to eventually add light/dark mode and font scale as additional test parameters to avoid regressions.